04.ls/ls_command.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COL_COUNT = 3
+
+def main
+  all_files = find_all_files
+  visible_files = delete_hiddenfile(all_files)
+  display_files(visible_files)
+end
+
+def find_all_files
+  Dir.entries('.').sort
+end
+
+def delete_hiddenfile(files)
```

Comment:
deleteだとfilesの中身を破壊的に変更する印象を受けるので、Rubyのメソッド名に合わせて

```suggestion
def reject_hidden_files(files)
```

とした方がいいかも。
（あと、hiddenとfileは2単語ですし、隠しファイルも複数あることが多いはずなのでfileも複数形にする方がベターです）